networkx/algorithms/chordal.py

```diff
@@ -200,11 +202,35 @@ def chordal_graph_cliques(G):
     ... ]
     >>> G = nx.Graph(e)
     >>> G.add_node(9)
-    >>> setlist = nx.chordal_graph_cliques(G)
+    >>> cliques = [c for c in chordal_graph_cliques(G)]
+    >>> cliques[0]
+    frozenset({1, 2, 3})
```

Comment:
I think this is okay because, though the ordering of the cliques is arbitrary, it should be *consistent* between function calls. If we wanted to be extra sure though, we could use `sorted` to guarantee the ordering, e.g.

```suggestion
    >>> cliques = sorted(chordal_graph_cliques(G))
    [frozenset({1, 2, 3}),
     frozenset({2, 3, 4}),
     frozenset({3, 4, 5, 6}),
     frozenset({7, 8}),
     frozenset({9})]
```

Reply:
This comment isn't a blocker though, we can always fix this up in a follow-up PR!